### PR TITLE
HW-47696 adding verificationStatus

### PR DESCRIPTION
**Pull request by kverma-hw**
https://jira.site1.hyperwallet.local/browse/HW-47696

### CODE REVIEW

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java [hunk 1]
```diff
@@ -23,8 +23,11 @@
 
     public static enum Status {PRE_ACTIVATED, ACTIVATED, LOCKED, FROZEN, DE_ACTIVATED}
 
+    public static enum VerificationStatus {EXPIRED, FAILED, UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED}
```
Comment (peter-joseph) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java]: The verification status in Portal Docs is not updated or this code exposes more of it? 
https://portal.hyperwallet.com/docs/api/v3/resources/users/retrieve

Please sync as needed

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 2]
```diff
@@ -186,6 +188,7 @@ public void testCreateUser_withProgramTokenInUserObject() throws Exception {
         assertThat(apiClientUser, is(notNullValue()));
         assertThat(apiClientUser.getFirstName(), is(equalTo("test-first-name")));
         assertThat(apiClientUser.getStatus(), is(nullValue()));
+        assertThat(apiClientUser.getVerificationStatus(), is(nullValue()));
```
Comment (peter-joseph) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: Please add a test case that validates that `verificationStatus` is not a null value and with any of the following values `enum VerificationStatus {EXPIRED, FAILED, UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED}`

File: src/test/java/com/hyperwallet/clientsdk/model/HyperwalletUserTest.java [hunk 3]
```diff
@@ -12,6 +14,7 @@ protected HyperwalletUser createBaseModel() {
         user
                 .token("test-token")
                 .status(HyperwalletUser.Status.ACTIVATED)
+                .verificationStatus(VerificationStatus.VERIFIED)
                 .createdOn(new Date())
                 .clientUserId("test-client-user-id")
                 .profileType(HyperwalletUser.ProfileType.INDIVIDUAL)
```
Comment (peter-joseph) [inline src/test/java/com/hyperwallet/clientsdk/model/HyperwalletUserTest.java]: Please add verification on the ff: 
1) validating `.verificationStatus(VerificationStatus.VERIFIED)` input from mocked request body 
2) validating  test for this code ` public HyperwalletUser clearVerificationStatus() {
        clearField("verificationStatus");
        verificationStatus = null;
        return this;
    }`

if it already exist please disregard this comment thanks

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java (hunk 1 above)
Comment (kverma-hw) [reply src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java]: Ash would be creating a separate ticket to get the document updated.

Comment (peter-joseph) [reply src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java]: 👍 cool
